mapbox/docs/guides/01-getting-started.md

```diff
@@ -14,13 +14,22 @@ After completing this guide, you will have your first Mapbox GL map with a CARTO
     </iframe>
 </div>
 
+
+### About MapboxGL license
```

Comment:
We could add that you can use v2 if you really need it (because you are using some new features) but it might cost you money.

Reply:
done